problems/finger_pointing_and_blaming.md

```diff
@@ -0,0 +1,48 @@
+---
+slug: finger-pointing-and-blaming
+stages:
+  - research
+short_description: Finger-Pointing and Blaming is the act of blaming someone for a problem instead of trying to fix or take accountability. This results in a toxic work culture where nobody feels safe and there's no trust.
+tags:
+  - finger-pointing
+  - blaming
+  - culture
+  - mentality
+  - trust
+keywords:
+  - team player
+  - problem
+  - satisfaction
+  - responsibilities
+  - leadership
+  - help
+  - performance
+  - environment
+  - efficiency
+  - empowerment
+---
+
+# Finger-Pointing And Blaming
+
+**TL;DR**
+
+Finger-Pointing and Blaming is the act of blaming someone for a problem instead of trying to fix or take accountability. This results in a toxic work culture where nobody feels safe and there's no trust. It is important to combat this problem quickly in order to establish a healthy and a more efficient work environment. Read more to find more about this!
+
+## Impact Of The Blame Culture
+
+Watching someone pawn their failures off or blame others for their mistakes is contagious and detrimental, particularly in the workplace. It may seem inconsequential but in an organization where blame is the norm, people are likely to be less creative, perform poorly, and mirror damaging behaviors to protect their own self-image.
```

Comment:
I would suggest rewriting this part in your own words. We use plagiarism checks and this part has strong resembles towards https://aaplonline.com/workplace-bullying-beyond-finger-pointing-to-a-culture-of-respect/.